lab_1/main.py

```diff
@@ -51,7 +92,15 @@ def create_language_profile(language: str, text: str, stop_words: list) -> dict
     :param stop_words: a list of stop words
     :return: a dictionary with three keys – name, freq, n_words
     """
-    pass
+
+    if not (isinstance(language, str) and isinstance(text, str) and isinstance(stop_words, list)):
+        return None
+    tokens = tokenize(text)
+    tokens = remove_stop_words(tokens, stop_words)
+    freq_dict = calculate_frequencies(tokens)
```

Comment:
you can transform tokens in one line, or at least tokenize and remove stop_words